assets/scripts/oppia-player-0.0.0.js

```diff
@@ -219,14 +219,14 @@
 
     var tagId = oppiaNode.getAttribute('id') || generateNewRandomId();
 
-    if (!height || height == 'auto') {
+    if (!height || height === 'auto') {
```

Comment:
Could you revert these? We don't really want to change this, and the inclusion of these in linting was a mistake. If you merge from develop again, the linter should work now.